Update Anycubic i3 Mega (S) profile

slghb opened the pull request:
- revert retraction settings as previous change causes issues with stringing.
  See also the discussion at https://github.com/prusa3d/PrusaSlicer-settings/pull/131#issuecomment-862191690
- sync many settings with Prusa MK2 profile and some with Prusa MK3 profile.
- remove no longer needed SUPPORTED print settings. Support settings can now be selected directly in the PrusaSlicer menu.
- add DAS FILAMENT PETG profile.

I tested these changes with good results on our printer. @Igami, feel free to test and please let me know if you see any issues with these changes.

Igami commented:
I will test it next weekend.

I have adapted the settings to the latest prusa settigs aka. MK3.

I've started a review, but need to find some time to pass it.

slghb commented on resources/profiles/Anycubic.ini:
```diff
@@ -825,37 +825,33 @@ end_gcode = M117 Cooling down...\nM104 S0 ; turn off extruder\nM107 ; Fan off\nM
 first_layer_height = 0.2
 first_layer_speed = 20
 gap_fill_speed = 40
-gcode_comments = 1
```
Prusa default is 0, so keep it disabled for now. Can be enable individually if needed.

slghb commented on resources/profiles/Anycubic.ini:
```diff
@@ -864,21 +860,25 @@ skirts = 1
-support_material_contact_distance = 0.1
+support_material = 0
 support_material_extrusion_width = 0.35
+support_material_buildplate_only = 0
```
Prusa default also is 0. Prusa Slicer also has a drop down menu in the Plater tab that let's you select "None", "Support on build plate only", "For support enforcers only" and "Everywhere". IMHO this should be used for enabling supports. We probably should also remove all the "SUPPORTED" entries from the print settings, as they are not longer needed with this menu. Do you have any comments on this?

slghb replied:
I interpret this thumps up emoji as agreement, so I will remove the SUPPORTED entries with my next update.

slghb replied:
I did this now.

slghb commented on resources/profiles/Anycubic.ini:
```diff
@@ -922,19 +939,23 @@ inherits = *0.20mm_mega*;*supported_mega*
 external_perimeter_extrusion_width = 0.6
 external_perimeter_speed = 35
 extrusion_width = 0.5
-fill_pattern = grid
```
I copied cubic form the MK2 "FAST" profile and would like to use it here because it prints faster than gyroid and also is a three dimensional pattern. Printing time between cubic and grid is quite similar.

Igami replied:
this profile is called QUALITY so it suggests that it is like the Prusa QUALITY profiles
I have taken some values from the newest MK3S+ profiles

please create an additional FAST profile

Something about infills:
https://blog.prusaprinters.org/everything-you-need-to-know-about-infills_43579/

slghb replied:
I think in the current state with this patch is that all profiles that are called QUALITY also get `fill_pattern = gyroid`. This change here should only impact the profile that's called DARFT.
But maybe it might make more sense to put this not to the `*0.30mm_mega*` template, but in the `0.30mm DRAFT  @MEGA` section, although in the end there will be no visible difference to the users.

Igami replied:
ah, it was my fault :facepalm: 
GitHub has hide the lines 913 to 924 so I thought it would impact the ```0.20mm QUALITY @MEGA``` profile.

grid is fine for the draft profile

slghb replied:
Yeah, I agree the github diff display is not really user-friendly here. I'm closing this as I think everything is resolved now.

slghb commented on resources/profiles/Anycubic.ini:
```diff
@@ -864,21 +860,25 @@ skirts = 1
+support_material = 0
 support_material_extrusion_width = 0.35
+support_material_buildplate_only = 0
+support_material_contact_distance = 0.2
```
I thought I copied this from the Prusa profile, but it seems the Prusa printers also have 0.1, so I might have mixed something here. I changed it back to 0.1 now.

slghb commented:
Sorry for my late reply, I was busy with other tasks for the last weeks. I applied most of your suggestions and left comments for the remaining.

slghb commented:
I think everything is resolved now, if not please let me know.

I now removed the supported profiles as discussed, as they're no longer needed with the latest PrusaSlicer.

I also moved the `solid_infill_speed` and `top_infill_extrusion_width` settings to the common part, as they were equal for all layer height settings.

rtyr commented:
Is it final, can we merge it?

slghb commented:
Yes, from my side it's ready and can be merged. I assume Igami's thumbs up reaction to my last comment also means Igami has no more open issues.
